developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
+|                               |                                                  | | R: Shareable                                                           |
+|                               |                                                  | | M: Mounted                                                             |
+|                               |                                                  | | G: Readable                                                            |
+|                               |                                                  | | D: Deletable                                                           |
+|                               |                                                  | | NV: Updateable, Renameable, Moveable                                   |
+|                               |                                                  | | W: Updateable (file)                                                   |
+|                               |                                                  | | CK: Creatable                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <nc:mount-type />             | The type of mount.                               | | ``''`` = local                                                         |
+|                               |                                                  | | ``'shared'`` = received share                                          |
+|                               |                                                  | | ``'group'`` = group folder                                             |
+|                               |                                                  | | ``'external'`` = external storage                                      |
+|                               |                                                  | | ``'external-session'`` = external storage                              |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <nc:is-encrypted />           | Whether the folder is end-to-end encrypted.      | | ``0`` for ``false``                                                    |
+|                               |                                                  | | ``1`` for ``true``                                                     |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:tags />                   | List of user specified tags.                     |                                                                          |
```

Comment:
Looking at https://github.com/nextcloud/server/blob/master/apps/dav/lib/Connector/Sabre/TagsPlugin.php and https://github.com/nextcloud/server/blob/master/apps/dav/lib/Connector/Sabre/TagList.php shows that the value is also xml. Can you put down the format in the example?